src/main/java/com/dre/brewery/configuration/sector/capsule/ConfigRecipe.java

```diff
@@ -17,20 +37,23 @@
 @AllArgsConstructor
 public class ConfigRecipe extends OkaeriConfig {
 
+    // Added because our configs don't support external comments yet
+    private Boolean enabled;
+
     private String name;
     private List<String> ingredients;
 
     @CustomKey("cookingtime")
-    private int cookingTime;
+    private Integer cookingTime;
     @CustomKey("distillruns")
-    private int distillRuns;
+    private Integer distillRuns;
     @CustomKey("distilltime")
-    private int distillTime;
-    private int wood;
-    private int age;
+    private Integer distillTime;
+    private Object wood; // int or String(Enum<BarrelWoodType>)
```

Comment:
We could probably use an okaeri serdes for things like this.

Reply:
I like that better than using `Object`. I'll probably add that to this PR or possibly after merge